src/fidesops/service/connectors/saas_query_config.py

```diff
@@ -23,6 +23,36 @@ def __init__(self, node: TraversalNode, endpoints: Dict[str, Endpoint]):
         super().__init__(node)
         self.endpoints = endpoints
 
+    @staticmethod
+    def _build_request_body(  # pylint: disable=R0913
+        custom_body: Optional[str],
```

Comment:
Does marking a kwarg `Optional[...]` suggest it should have a default value? It's confusing here since the kwargs with default values of `None` aren't marked `Optional`

Reply:
You're right, variables with default of `None` should be `optional`. I'll update!

Reply:
And vice versa, `custom_body` shouldn't be `Optional` unless this method could work with `custom_body` as `None`

Reply:
I've updated `custom_body` to default to `None`, since I want to keep err handling within this method